microsoft/testsuites/core/floppy.py

```diff
@@ -30,6 +37,12 @@ class Floppy(TestSuite):
         priority=1,
     )
     def check_floppy_module(self, node: RemoteNode) -> None:
+        os_version = node.os.information.version
+        is_centos = isinstance(node.os, CentOs)
+        if is_centos and os_version <= "7.7.0":
+            raise SkippedException(
+                "CentOS < 7.7 are not receiving fixes for this issue."
```

Comment:
The comments should be "<=7.7" as well. But how about "7.7.1"? Does `< "7.8.0"` is more accurate?